### ISSUE TITLE
Handle fonts from `~/.local/share/fonts`

### ISSUE THREAD

**Issue by sduensin** [ONLYOFFICE/DocumentBuilder#35]
Given the following document:

![Screenshot from 2020-04-30 17-11-12](https://user-images.githubusercontent.com/98645/80764210-cd14a100-8b05-11ea-9218-0f7be97d7f14.png)

And this document script:
```
builder.OpenFile("Manual.docx");
builder.SaveFile("pdf", "/tmp/Manual.pdf");
builder.CloseFile();
```
The resulting PDF does not contain the custom font:

![Screenshot from 2020-04-30 17-11-51](https://user-images.githubusercontent.com/98645/80764261-e87fac00-8b05-11ea-909f-8bb5fc617ded.png)

I am running Linux Mint 19.3 Cinnamon.

**Comment by ShockwaveNN** [comment]
Hi, could you attach original docx file?

**Comment by sduensin** [comment]
Sure!  Test document and font attached.

[FontProblem.docx](https://github.com/ONLYOFFICE/DocumentBuilder/files/4572119/FontProblem.docx)

[BreatheFire-65pg.ttf.zip](https://github.com/ONLYOFFICE/DocumentBuilder/files/4572127/BreatheFire-65pg.ttf.zip)

**Comment by ShockwaveNN** [comment]
Seems everything fine for me:
![image](https://user-images.githubusercontent.com/668524/81146775-9ca78980-8f81-11ea-8fb7-a2e1ea8d0625.png)

Which version of DocumentBuilder are you using?
And in which plase this font is installed? I installed mine at (via default Kubuntu font utility) at:
```
fc-list | grep Breath
/usr/local/share/fonts/b/BreatheFire_65pg.ttf: Breathe Fire:style=Regular
```

**Comment by sduensin** [comment]
```
$ fc-list | grep Breath
/home/scott/.local/share/fonts/BreatheFire-65pg.ttf: Breathe Fire:style=Regular
```
```
$ dpkg -l | grep documentbuilder
ii  onlyoffice-documentbuilder                 5.3.3-43                                         amd64        tool to create documents in office file formats including Office Open XML formats:
```

I installed the font using the "Fonts" application in Linux Mint.

**Comment by ShockwaveNN** [comment]
@sduensin try to copy this font to same location as mine and see if it fix your problem

**Comment by sduensin** [comment]
That fixed it.  Still seems like a bug if the system itself knows about the font in my home directory.  

Thanks!  I'm back in business!  :-)

**Comment by ShockwaveNN** [comment]
@sduensin Yeah, I agree that it is a bug, and seems it should be fixed in v5.5, but we skipped several releases of DocumentBuilder for some reason unknown for me, so I'll try to figure out can we release v5.5

**Comment by ShockwaveNN** [comment]
DocumentBuilder v6.0.0 is released

This issue should be fixed
